submission-scripts/script-templates/magpie-hive

```diff
@@ -0,0 +1,80 @@
+#############################################################################
+# Hive configurations
+#############################################################################
+#
+# Should Hive be run
+#
+# Specify yes or no.  Defaults to no.
+#
+export HIVE_SETUP=yes
+
+# Version
+
+export HIVE_VERSION="HIVEVERSIONDEFAULT"
+
+export HIVE_PORT=HIVEDEFAULTPORT
+
+# Path to your Hive build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+# Ensure the build matches the Hadoop/HDFS version this will run against.
+#
+export HIVE_HOME="HIVEDIRPREFIX/base/apache-hive-${HIVE_VERSION}-bin"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp.  This will store local conf files and log files for your
+# job.  If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option.  See README for more details.
+
+export HIVE_LOCAL_DIR="LOCALDIRPREFIX/hive"
+
+# Hive Clients will make a connection to a metastore server which in turn queries the datastore (PostgreSQL)
+# for metadata. This sets the location of the metastore
+export HIVE_METASTORE_DIR="LUSTREDIRPREFIX/hive/database/postgres/data"
+
+# Configurations for Postgres, used as a backend for HiveServer/Hive Metastore
+export POSTGRES_HOME="POSTGRESDIRPREFIX/base/pgsql"
+export PGPORT=POSTGRESDEFAULTPORT
+export PGDATA="LUSTREDIRPREFIX/hive/database/postgres/data"
+
+# DB name for this project
+export HIVE_DB_NAME="HIVEDEFAULTDBNAME"
+
+###########################################################################
+# Hive Job/Run Configurations
+############################################################################
+
+# Set hive job for MAGPIE_JOB_TYPE = hive
+# Use 'testbench' to run benchmark testing for Hive via hive-testbench modules
+#
+# export HIVE_MODE="testbench"
```

Comment:
this shouldn't be uncommented by default, as it is checked in magpie-check-inputs as always required to be set